tests/fixtures/assignment.json

```diff
@@ -379,5 +379,41 @@
 		"data": {
 			"no_url": "bad_url_failure"
 		}
+	},
+	"provisional_grades_bulk_select" : {
+		"method": "PUT",
+		"endpoint": "courses/1/assignments/1/provisional_grades/bulk_select"
+	},
```

Comment:
I think we're skipping `bulk_select`? If so, delete this.

It's worth noting that this needs a `"data"` section to function properly.